### PR TITLE
US102400 - Add quick-eval image for usage in image based nav

### PR DESCRIPTION
**Pull request by mtjandra**
QuickEval

* Adding this image here as instructed by [US102400](https://rally1.rallydev.com/#/42960320374d/detail/userstory/271013159028)
* Later I will need to figure out how to get the navigation to use this image

### CODE REVIEW

**Comment by svanherk** [comment]
All the other image-based nav icons seem to be 60px by 60px.  Are they resizable in the monolith or taken as is?

**Comment by mtjandra** [comment]
> All the other image-based nav icons seem to be 60px by 60px. Are they resizable in the monolith or taken as is?

I see, is it `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 59.754 60">` that you are concerned about? I go circle back with the designer to double check

Also @edwardg8 mentioned it would probably be nicer to view not all in one line. So I will modify to add newlines. I was just putting what the designer gave me

**Comment by edwardg8** [comment]
Also make sure every image has width, height, viewBox attributes on them. I think i.e11 (no longer supported) and Edge have problem with SVG with these attributes missing.

File: nav-icons/quick-eval.svg [hunk 1]
```diff
@@ -1 +1,30 @@
...
\ No newline at end of file
+<svg width="59.754" height="60" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 59.754 60">
+  <defs>
+    <style>.cls-1{isolation:isolate;}.cls-2{fill:#2dc1d7;}.cls-10,.cls-3,.cls-7{fill:#fff;}.cls-3{opacity:0.4;mix-blend-mode:lighten;}.cls-4{fill:#9ac866;}.cls-12,.cls-4,.cls-8{stroke:#575b5d;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px;}.cls-5{fill:#fded59;}.cls-10,.cls-6{opacity:0.5;}.cls-8{fill:none;}.cls-9{fill:#ec7139;}.cls-11{fill:#add8a6;}.cls-12{fill:#fabf2e;}.cls-13{fill:#575b5d;}</style>
+  </defs>
+  <g class="cls-1">
+    <g id="Layer_2" data-name="Layer 2">
```
Comment (edwardg8) [inline nav-icons/quick-eval.svg]: nit: probably dont need `data-name="Layer 2"`

Comment (mtjandra) [reply nav-icons/quick-eval.svg]: Removed

**Comment by mtjandra** [comment]
From Sara:

>sounds like it wasn't exported with it's artboard. It's ok for it not to be bang on since it's vector from a visual perspective, changing it in the code should be ok too

I have updated for it to be 60x60 thanks!

**Pull request merged**